examples/step-50/step-50.cc

```diff
@@ -52,8 +52,8 @@
 
 #include <deal.II/lac/generic_linear_algebra.h>
 
-// Comment the following \#define if you have PETSc and Trilinos installed and
-// you prefer using PETSc in this example:
+// Comment the following `\#define` in or out if you have PETSc and
```

Comment:
This way to escape seems to be incorrect.

Reply:
Yep, already fixed.